server/metrics/metrics.go

```diff
@@ -109,6 +111,18 @@ func NewMetrics(info InstanceInfo) Metrics {
 	)
 	m.registry.MustRegister(m.apiTime)
 
+	m.messageSyncTime = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace:   MetricsNamespace,
+			Subsystem:   MetricsSubsystemApp,
+			Name:        "message_sync_time",
+			Help:        "Time to sync the messages",
+			ConstLabels: additionalLabels,
+		},
+		[]string{"source"},
```

Comment:
I don't love the "source" label name, it is difficult to know exactly what it means, I would prefer something like `msteams_event_type` that is self explanatory.

Reply:
Nevermind, I saw that is not what I was thinking. Ok, source I guess is ok here.